src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml

```diff
@@ -2810,7 +2825,7 @@ well as menu structures (for main menu and popup menus).
 
    <cmd id="historyDismissContext"
         label="Dismiss History Context"
-        buttonLabel="&#x00AB; Back"
```

Comment:
This `&#x00AB;` does show up in the UI, in the History Pane, as the little "<<Back" button. Ideally we'd keep it.

![Screen Shot 2021-09-28 at 2 34 31 PM](https://user-images.githubusercontent.com/10569626/135169185-36dda724-f39c-40e7-b0b2-03ed7fe09971.png)

Reply:
I'll look more into this! Right now the script wants to translate it to an illegal character which makes the compiler unhappy